core/src/main/java/org/arquillian/cube/impl/client/container/ContainerConfigurationController.java

```diff
@@ -64,12 +67,20 @@ public void remapContainer(@Observes BeforeSetup event, CubeRegistry cubeRegistr
                     int containerPort = getDefaultPortFromConfigurationInstance(newConfigurationInstance,
                             configurationClass, configurationClassPortField);
 
-                    PortBinding bindingForExposedPort = null;
-                    if ((bindingForExposedPort = binding.getBindingForExposedPort(containerPort)) != null) {
+                    PortBinding bindingForPort = null;
+                    if ((bindingForPort = binding.getBindingForExposedPort(containerPort)) != null) {
```

Comment:
Yep.  That's a good interim solution.  One quick question: what's the difference between bindings() and configuredBindings()?  I added a comments in my latest commit, but I'd like to verify that those comments are correct, i.e. bindings() are the actual bindings for the running container and configuredBindings() are simply what's been configured, e.g. container ports in the pod.json, or EXPOSE elements of a Dockerfile.